libr/anal/d/cc-avr-8

```diff
@@ -0,0 +1,26 @@
+default.cc=avr
+
+avr=cc
+cc.avr.name=avr
+cc.avr.argn=r26
```

Comment:
Yeah, I'm still a bit confused by that grammar, sorry :-S

Not sure how to best implement the two slides from #5452 there since there are quite particular rules about which bytes end up where (please take a quick look at #5452... can those be implemented with the current grammar?).